<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatGroup</class>
 <widget class="QWidget" name="ChatGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
     <item>
      <widget class="QTextEdit" name="group_chat_textEdit"/>
     </item>
     <item>
      <widget class="QListWidget" name="group_chat_listWidget"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="group_chat_lineEdit">
       <property name="minimumSize">
        <size>
         <width>520</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="send_infoButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>50</height>
        </size>
       </property>
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
